front-end/src/components/Post/GovernanceSideBar/index.tsx

```diff
@@ -43,7 +43,7 @@ interface Props {
 	startTime: string
 }
 
-const GovenanceSideBar = ({ canEdit, className, isMotion, isProposal, isReferendum, isTipProposal, isTreasuryProposal, onchainId, onchainLink, startTime, status }: Props) => {
+const GovernanceSideBar = ({ canEdit, className, isMotion, isProposal, isReferendum, isTipProposal, isTreasuryProposal, onchainId, onchainLink, startTime, status }: Props) => {
```

Comment:
This is the default and only export so it wouldn't break things. But please fix the typo in the file references too.

Reply:
Done